main/design_documents/ARM_NEON_SVE_BRIDGE.rst

```diff
@@ -0,0 +1,78 @@
+=======================================
+Design Document for ARM_NEON_SVE_BRIDGE
+=======================================
+
+This document describes the design of the intrinsics defined in
+`NEON_SVE Bridge <../acle.rst#neon-sve-bridge>`_. These C intrinsics enable the
+conversion between NEON and SVE vectors with no performance overhead, and
+duplicating NEON vectors across SVE vectors.
+
+Location
+########
+
+The intrinsics are defined in ``arm_neon_sve_bridge.h``. If
+``arm_neon_sve_bridge.h`` is available on the compiler,
+``__ARM_NEON_SVE_BRIDGE`` is defined as 1. Users include the NEON/SVE Bridge
+as follows:
+
+.. code:: c
+
+    #ifdef __ARM_NEON_SVE_BRIDGE
+    #include "arm_neon_sve_bridge.h"
+    #endif //__ARM_NEON_SVE_BRIDGE
+
+``arm_neon_sve_bridge.h`` includes ``arm_neon.h`` and ``arm_sve.h``.
+
+Suggestions & Considerations
+############################
+
+Why not enable the extraction of any 128-bit subvector in an SVE vector?
+------------------------------------------------------------------------
+
+Extracting other 128-bit subvectors goes against the no-op design.
+This functionality is already available with the ACLE intrinsic 
+``svdupq_lane``. For example:
+
+``svuint64_t sve_vector_a = ...;``
+
+``uint64x2_t neon_vec = svget_neonq_u64(svdupq_lane_u64(sve_vector_a, 1));``
+
+Why not enable the insertion of a 128-bit NEON vector into a specific SVE subvector?
+------------------------------------------------------------------------------------
+
+arm_sve.h does not currently provide any intrinsics for inserting individual
+elements into a specific lane of an SVE vector. The approach should be
+consistent between arm_sve.h and arm_neon_sve_bridge.h
```

Comment:
Nit: elsewhere these header files (across the above three lines) appear in double backticks.